apps/api-server/src/routes/widget/widget.js

```diff
@@ -0,0 +1,187 @@
+const express = require('express');
+const bruteForce = require('../../middleware/brute-force');
+const db = require('../../db');
+const fs = require('fs');
+const config = require('config');
+const path = require('path');
+const createError = require('http-errors');
+const flattenObject = require('../../util/flatten-object');
+const widgetSettingsMapping = require('./widget-settings');
+const reactCheck = require('../../util/react-check');
+
+let router = express.Router({ mergeParams: true });
+
+// brute force
+router.use(bruteForce.globalMiddleware);
```

Comment:
Deze staat hier goed, maar weet dat die functie momenteel niets doet...

Reply:
Ah, scherp! Ik zal 'em voor nu toch laten staan, mochten we de global brute force weer willen activeren.